bitnami/thanos/Chart.yaml

```
@@ -28,4 +28,4 @@ name: thanos
 sources:
   - https://github.com/bitnami/bitnami-docker-thanos
   - https://thanos.io
-version: 5.2.7
+version: 5.3.0
```

Add a new line character at the end of the file to avoid the linter to complain:
```suggestion
version: 5.3.0

```